plugins.sh

```diff
@@ -40,3 +40,6 @@ do
 done
 
 rm -f /tmp/pluginList.txt
+
+chown root:root /opt/sonarqube -R
```

Comment:
Let's move these to the end of the Dockerfile where folks would expect to find this type of thing